great-idea/css/index.css

```diff
@@ -63,4 +63,202 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* General Styling */
+
+body {
+	width: 80vw;
+	margin: 0 auto;
+}
+
+ul,li,img,div {
+	display: inline-block;
+}
+
+h1 {
+	font-size: 4em;
+	text-align: center;
+}
+
+h2 {
+	font-size: 1.5em;
+}
+
+p {
+	font-size: 18px;
+	line-height: 1.4;
+}
+
+
+/* Header Styling */
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	width: 80vw;
+	margin-bottom: 50px;
+	margin-top: 15px;
+}
+
+ul {
+	display: flex;
+	justify-content: space-between;
+	width: 70%;
+}
+
+a {
+	text-decoration: none;
+	color: #bbb;
+}
+
+.title {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+
+.title-container {
+	text-align: center;
+}
+
+.button {
+	border: 1px solid #000;
+	padding: 10px 0 10px 0;
+	width: 70%;
+	margin: 0 auto;
+}
+
+.services-header-img {
+	width: 100%;
+	margin-bottom: 50px;
+}
+
+.services-title {
+	text-align: left;
+	font-size: 1.5em;
+}
+
+/* Main Content Styling */ 
+
+main {
+	border-top: 1px solid #000;
+	border-bottom: 1px solid #000;
+	margin-top: 50px;
+}
+
+.top-container {
+	display: flex;
+	justify-content: space-between;
+	margin-top: 50px;
+}
+
+.top-content {
+	width: 47%;
+}
+
+.lower-container {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 50px;
+}
+
+.lower-content {
+	width: 30%;
+}
+
+.middle-img {
+	width: 100%;
+	margin: 50px auto;
+}
+
+.services-main {
+	display: flex;
```

Comment:
Maybe consider splitting the CSS for the services page into its own `services.css` file for increased readability and maintainability?